howto/container/launch.md

```diff
@@ -53,9 +53,9 @@ By default, every container is scheduled by AMS onto a LXD node. Alternatively,
 
 ## Launch a container with a different Anbox platform
 
-By default, every container starts with the `null` platform (see [Anbox platforms](https://discourse.ubuntu.com/t/anbox-platforms/18733)). The selected platform cannot be changed at runtime and must be selected when the container is created. For example, you can launch a container with the `swrast` platform like this:
+By default, every container starts with the `null` platform (see [Anbox platforms](https://discourse.ubuntu.com/t/anbox-platforms/18733)) and with the `webrtc` platform when `--enable-graphics` is specified. The selected platform cannot be changed at runtime and must be selected when the container is created. For example, you can launch a container with the `webrtc` platform like this:
 
-    amc launch -p swrast <application-id>
+    amc launch -p webrtc <application-id>
```

Comment:
Wouldn't that create confusion ?
We essentially have two ways of settings the `webrtc` platform: `--enable-graphics` and `--platform=webrtc`.
Considering we only have two options (and the other option - `null` - is the default one), users don't really need the `--platform` flag. So would it make sense to push the concept of platform into a more "advanced" section ? We could only expose the `--enable-graphics` without introducing the complexity of platforms